pywr/solvers/cython_glpk.pyx

```diff
@@ -47,16 +47,27 @@ cdef class CythonGLPKSolver:
     cdef list aggregated
     cdef public object stats
 
+    # Internal representation of the basis for each scenario
+    cdef int[:, :] row_stat
+    cdef int[:, :] col_stat
+    cdef bint is_first_solve
+    cdef bint has_presolved
+    cdef bint use_presolve
```

Comment:
It would be good to expose this as an option in JSON. We already have a "solver" section.

```
"solver": {
    "name": "glpk",
    "presolve": true
}
```

I don't know if it should default on or off. I remember trying it in the past and found it was slower or no difference with presolving.